microsoft-edge/extensions/update/api/using-addons-api.md

```diff
@@ -553,7 +553,7 @@ The following code is an example of a PowerShell script that uses the REST API t
 
 To use this script, fill in the `$ClientID`, `$ClientSecret`, `$ProductID`, and `$FilePATH` values at the top of the script.
 
-```powershell
+```shell
```

Comment:
should this say specifically PowerShell?
@leahmsft 

review rendered of file
microsoft-edge/extensions/update/api/using-addons-api.md
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions/update/api/using-addons-api?branch=pr-en-us-3477

does gh preview render lang name reliably?
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/triple-backticks/microsoft-edge/extensions/update/api/using-addons-api

Reply:
reverted, this is a p.s. script